vars/infra.groovy

```diff
@@ -272,23 +272,13 @@ void prepareToPublishIncrementals() {
 void maybePublishIncrementals() {
     if (new InfraConfig(env).isRunningOnJenkinsInfra() && currentBuild.currentResult == 'SUCCESS') {
         stage('Deploy') {
-          timeout(300) {
-            node('maven || linux || windows') {
-                timeout(15) {
-                    withCredentials([string(credentialsId: 'incrementals-publisher-token', variable: 'FUNCTION_TOKEN')]) {
-                        if (isUnix()) {
-                            sh '''
-    curl --retry 10 --retry-delay 10 -i -H "Authorization: Bearer $FUNCTION_TOKEN" -H 'Content-Type: application/json' -d '{"build_url":"'$BUILD_URL'"}' "https://incrementals.jenkins.io/" || echo 'Problem calling Incrementals deployment function'
-                            '''
-                        } else {
-                            bat '''
-    curl.exe --retry 10 --retry-delay 10 -i -H "Authorization: Bearer %FUNCTION_TOKEN%" -H "Content-Type: application/json" -d "{""build_url"":""%BUILD_URL%""}" "https://incrementals.jenkins.io/" || echo Problem calling Incrementals deployment function
-                            '''
-                        }
-                    }
-                }
-            }
-          }
+            httpRequest url: 'https://incrementals.jenkins.io/',
+                        httpMode: 'POST'
+                        contentType: 'APPLICATION_JSON',
+                        validResponseCodes: '100:599',
+                        timeout: 300,
+                        requestBody: '{"build_url":"'$BUILD_URL'"}',
+                        authentication: 'incrementals-publisher-token'
```

Comment:
Does this actually work? This will be a `Bearer` token, not username/password. https://www.jenkins.io/doc/pipeline/steps/http_request/#httprequest-perform-an-http-request-and-return-a-response-object is not specific.

Reply:
no it doesn't work, it requires username / password